.circleci/config.yml

```diff
@@ -164,7 +164,7 @@ commands:
             - deps-{{ .Branch }}-{{ checksum ".pnp.cjs" }}
       - run:
           working_directory: integration-tests
-          command: yarn install --immutable-cache
+          command: yarn install --immutable
```

Comment:
just confirming that this change still lets the cache of all the other packages be leveraged (locally and in CI), correct?

Reply:
> | Definition | Description |
> | --- | --- |
> | --immutable​ | Abort with an error exit code if the lockfile was to be modified |
> | --immutable-cache​ | Abort with an error exit code if the cache folder was to be modified |
> | ... | ... |

https://yarnpkg.com/cli/install/#options